src/interceptors/ClientRequest/MockHttpSocket.ts

```diff
@@ -205,6 +206,13 @@ export class MockHttpSocket extends MockSocket {
       return
     }
 
+    // Return fake address information for the socket.
+    this.address = () => ({ 
+      address: '0.0.0.0',
```

Comment:
What do you think about setting `127.0.0.1` as the fake address? Seeing localhost can hint people that this address is, indeed, mocked.